Update Master Forge.rst

duianto opened the pull request:
fixed links
some rewrites
added missing pre-order items
fixed kickstarter bundle ergo bolt-on set amount

CCMaverick commented on docs/Master Forge.rst:
```diff
@@ -181,14 +209,14 @@ the first time, it’s important to make sure that either the :ref:`electrical b
+As a rule of thumb, all Forge :doc:`anchor bodies<Anchor Bodies>`, including the Master Forge Digitizers, must receive power through their front left USB-C port. It's important to use that specific port to connect your device to your computer because no other port will permit your Forge to function correctly. The other three ports are outgoing ports in order to connect other anchor bodies and :do ...
 
-If you haven't done so, now would be the time to plug the included USB-C to USB-A cable included with your order into the LEFT :doc:`digitizer<Master Forge:The Digitizers>`. If you would rather use an after-market USB-C to USB-C cable instead, due to a personal preference or computer requirement, that is also okay. Regardless of your selection, we'll refer to the cable that connects directly to t ...
+If you haven't done so, now would be the time to plug in the included USB-C to USB-A cable into the front LEFT :doc:`digitizer<Master Forge:The Digitizers>`. If you would rather use an after-market USB-C to USB-C cable instead, due to a personal preference or computer requirement, that is also okay. Regardless of your selection, we'll refer to the cable that connects directly to the computer as t ...
```
Remove "in" from the first sentence. The structure is essentially "plug the cable into the computer," and rewriting as "plug in the cable into the computer" is redundant.

CCMaverick commented on docs/Master Forge.rst:
```diff
@@ -432,12 +468,16 @@ It will look something like this:
-CharaChorder One. In the example above, you would press ``K`` for
-Keyboard, ``M`` for Mouse, ``C`` for Chording, ``D`` for Display, and
-``R`` for Resources.
+CharaChorder One. In the example above, you would press:
```
This should say "Master Forge" not "CharaChorder One"

CCMaverick reviewed (changes requested):
Comments left next to the places that need changes. I think there are only like 3 or 4

CCMaverick commented on docs/Master Forge.rst:
```diff
@@ -181,14 +209,14 @@ the first time, it’s important to make sure that either the :ref:`electrical b
+As a rule of thumb, all Forge :doc:`anchor bodies<Anchor Bodies>`, including the Master Forge Digitizers, must receive power through their front left USB-C port. It's important to use that specific port to connect your device to your computer because no other port will permit your Forge to function correctly. The other three ports are outgoing ports in order to connect other anchor bodies and :do ...
 
-If you haven't done so, now would be the time to plug the included USB-C to USB-A cable included with your order into the LEFT :doc:`digitizer<Master Forge:The Digitizers>`. If you would rather use an after-market USB-C to USB-C cable instead, due to a personal preference or computer requirement, that is also okay. Regardless of your selection, we'll refer to the cable that connects directly to t ...
+If you haven't done so, now would be the time to plug in the included USB-C to USB-A cable into the front LEFT :doc:`digitizer<Master Forge:The Digitizers>`. If you would rather use an after-market USB-C to USB-C cable instead, due to a personal preference or computer requirement, that is also okay. Regardless of your selection, we'll refer to the cable that connects directly to the computer as t ...
```
Not sure if you're agreeing with my recommendation or not lol

duianto replied:
Agreeing. 😄

CCMaverick commented on docs/Master Forge.rst:
```diff
@@ -492,7 +532,7 @@ By default, auxiliary layers are accessible by pressing and holding the "layer a
     #. Make sure that the A2 layer access key is mapped to the key of your choice on the Alpha layer (A1 layer). This is the key you will use to toggle into the A2 layer
     #. On the A2 layer, find the same location that you mapped the A2 layer access key and change that key (on the A2 layer) to ``No Key Pressed``. This change is what prevents your device from shifting back into the A1 layer
-    #. On the A2 layer, choose another location and map the A2 layer access key there. This is the key that you will use to return to the Alpha layer.
+    #. On the A2 layer, choose another location and map the A1 layer access key there. This is the key that you will use to return to the Alpha layer (A1 layer).
```
Okay, with that explanation of future updates, I agree with ceasing to use alpha, numeric, and whatever else in favor of the letter+number combo

duianto committed:
Update Master Forge.rst

M4G:
fixed links
some rewrites
added missing pre-order items
fixed kickstarter bundle ergo bolt-on set amount

CC2:
added a target (and linked to from the M4G page): design the layout

duianto committed:
M4G, review suggestions

duianto committed:
M4G, rename layer Alpha to A1

duianto commented:
Ready for review
